app/presenters/notes_presenter.rb

```diff
@@ -4,6 +4,7 @@ def self.grouped_notes(client)
     # Generate notes from documents on-the-fly, so the summary can change as clients upload more documents.
     notes = (client.notes.includes(:user) + SystemNote.where(client: client))
     synthetic_notes = synthetic_notes_from_documents(client.documents.order(created_at: :asc))
+    synthetic_notes += synthetic_notes_from_outbound_calls(client.outbound_calls.order(created_at: :asc))
```

Comment:
why do we call them synthetic notes? 🤔

Reply:
Notes for documents and outbound calls are not saved into the database -- they're created as we load the notes information onto the page for display purposes only. This allows us to group together document information to show all docs uploaded on the same day as one message, and show the latest statuses of the phone call. Basically, we're creating the bodies for the notes on the fly